addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -1693,17 +1700,21 @@ export class Rtc {
     }
 
     /**
-     * @param {RtcSession} session
-     * @param {String} [videoType]
+     * @param {import("models").RtcSession} session
+     * @param {Object} [param1]
+     * @param {String} [param1.videoType]
+     * @param {String} [param1.cleanup]
      */
-    removeVideoFromSession(session, videoType = false) {
-        if (videoType) {
-            this.updateActiveSession(session, videoType);
-            closeStream(session.videoStreams.get(videoType));
-            session.videoStreams.delete(videoType);
+    removeVideoFromSession(session, { type = false, cleanup = true } = {}) {
+        if (type) {
+            this.updateActiveSession(session, type);
+            cleanup && closeStream(session.videoStreams.get(type));
```

Comment:
prefer

```js
if (conditional) {
   fn();
}
```
Over `conditional && fn()`.

1st code is more idiomatic and doesn't make eslint panic with a warning on the line.